tests/fcio_test_unknown_tags.c

```diff
@@ -14,6 +14,8 @@
 
 int main(int argc, char* argv[])
 {
+  assert(argc >= 2);
```

Comment:
Haven't checked if other tests do the same - but we should aim to avoid assert for functional checks.